.claude/skills/changelog-collect/SKILL.md

```
@@ -0,0 +1,328 @@
+
+```
+English:
+- Fix Select incorrect height when value is empty
+
+Chinese:
+- 修复 Select value 为空时高度不正确的问题
+```
+
+提取规则：
+
+- 识别 `🇺🇸 English` 或 `English:` 标记后的 `- ` 行作为英文描述
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)